var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -926,6 +928,29 @@ def cmake_args(self):
 
         # Semicolon seperated list of runtimes to enable
         if runtimes:
+            if self.spec.satisfies("@:14"):
+                # The older versions are not
+                # careful enough with the order of
+                # the runtimes.
+                # Instead of
+                # applying
```

Comment:
Can we squeeze these onto fewer lines? The line length limit in Spack is 99 chars.

Reply:
Sure. Done. I really did a sloppy job with this PR, sorry about that.